packages/core/deprecations/core-deprecations-server-internal/src/deprecations/i18n_texts.ts

```diff
@@ -106,6 +111,15 @@ export const getApiDeprecationsManualSteps = (details: RouterDeprecatedRouteDeta
       );
       break;
     }
+    case 'deprecate': {
+      manualSteps.push(
+        i18n.translate('core.deprecations.deprecations.manualSteps.removeTypeExplainationStep', {
+          defaultMessage:
+            'For now, the API will still work, but will be moved or removed in a future version. Check the documentation for more information. If you are no longer using the API, you can mark this issue as resolved. It will no longer appear in the Upgrade Assistant unless another call using this API is detected.',
```

Comment:
```suggestion
            'For now, the API will still work, but will be moved or removed in a future version. Check the Learn more link for more information. If you are no longer using the API, you can mark this issue as resolved. It will no longer appear in the Upgrade Assistant unless another call using this API is detected.',
```